docs/core/tools/dotnet-cleanup.md

```diff
@@ -0,0 +1,102 @@
+---
+title: dotnet cleanup command
+description: The dotnet cleanup command cleans up code formatting to match editorconfig settings for the current directory.
+ms.date: 07/12/2020
+---
+# dotnet cleanup
+
+**This article applies to:** ✔️ .NET Core 6.x SDK and later versions
+
+## Name
+
+`dotnet cleanup` - Cleans up code formatting to match editorconfig settings.
+
+## Synopsis
+
+```dotnetcli
+dotnet [options] cleanup [<PROJECT | SOLUTION>] [command]
+
+dotnet clean -h|--help
+```
+
+## Description
+
+`dotnet cleanup` is a code formatter for dotnet that applies style preferences to a project or solution. Preferences will be read from an .editorconfig file, if present, otherwise a default set of preferences will be used.
```

Comment:
Does this execute third party codefixes? Could the document be clear about this whether it's a yes or no?